### DISCUSSION TITLE
Feature Request

### DISCUSSION THREAD

**Discussion by ikeman32** [DraqueT/PolyGlot#1215] [Ideas]
Would like to be able to view the lexicon with conland and local lang in a side by side view. Presently it's one or the other.

**Comment by DraqueT** [comment]
The lexicon should display words from the corollary language to the right of their dictionary forms like below. Were you thinking of something fundamentally different than this, though?

![image](https://user-images.githubusercontent.com/8284425/193941574-f741496c-4aec-48db-bae4-04ad1f3dd966.png)
